project/index.html

```diff
@@ -17,8 +17,8 @@
         <script src="https://cdnjs.cloudflare.com/ajax/libs/ol3/4.0.1/ol.js"></script>
         <script src="https://cesiumjs.org/releases/1.17/Build/Cesium/Cesium.js"></script>
         <link rel="stylesheet" href="https://cesiumjs.org/releases/1.17/Build/Cesium/Widgets/widgets.css" />
-        <script src="libs/cesium-navigation/cesium-navigation.js"></script>
-        <link rel="stylesheet" href="libs/cesium-navigation/cesium-navigation.css" />
+        <!-- <script src="libs/cesium-navigation/cesium-navigation.js"></script>
```

Comment:
I think we should change these to point to MapStore2/web/client/libs/cesium-navigation ...